projects/demo/src/modules/components/link/link.component.ts

```diff
@@ -44,7 +44,9 @@ export class ExampleTuiLinkComponent {
 
     icon = '';
 
-    readonly iconVariants = ['tuiIconStarLarge', 'tuiIconMapPinLarge'];
+    readonly iconVariants = ['tuiIconStarLarge', 'tuiIconMapPinLarge', 'tuiIconMapPin'];
 
     iconAlign: TuiHorizontalDirection = this.iconAlignValues[0];
+
+    size = '1.5rem';
```

Comment:
nit: dont we have `rem(value)` wrap as `px(value)`?

Reply:
I think we don't. But in reality we would also bind `[style.--tui-link-icon-size.rem]`, so it doesn't matter. That's external, it's up to people using Taiga UI.